content/px-insights/key-platform-behaviors.md

```diff
@@ -0,0 +1,46 @@
+# Key Platform Behaviors
+
+## Asynchronous Processing
+
+The PX Insights API processes your reviews asynchronously. When you submit reviews to the `/api/v1/reviews/ingest/async` endpoint, they are placed in a queue for processing. This approach offers several benefits:
```

Comment:
We mention reviews specifically but we're in a more general section. What happens when we make other APIs public?